fuzz/README.md

```diff
@@ -0,0 +1,10 @@
+# Fuzzing with libfuzzer
```

Comment:
Might be worth having this also reference [the note in the crate README.](https://github.com/image-rs/image-extras/blob/f686d89c3aa53b22fcd23e52fc509f7c263fbc79/README.md?plain=1#L27)

Reply:
Done (linked to the README and repeated part of the note).